src/lib/pages/backoffice/Loan/LoanDetails/LoanScanCopy/LoanScanCopy.js

```diff
@@ -0,0 +1,104 @@
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import { Header, Icon, Message, Modal } from 'semantic-ui-react';
+import { ItemsSearch } from '../../../Actions/CheckIn/ItemsSearch';
+
+export default class LoanScanCopy extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      isLoading: false,
+      wrongBarcode: false,
+    };
+    this.isLoadingHandler = this.isLoadingHandler.bind(this);
+    this.wrongBarcodeHandler = this.wrongBarcodeHandler.bind(this);
+  }
+
+  isLoadingHandler() {
+    this.setState({
+      isLoading: true,
+    });
+  }
+
+  wrongBarcodeHandler() {
+    this.setState({
+      wrongBarcode: true,
+    });
+  }
+
+  closeModal = () => {
+    const { clearSearchBar } = this.props;
+    this.setState({
+      isLoading: false,
+      wrongBarcode: false,
+    });
+    this.props.onClose();
+    clearSearchBar();
+  };
+
+  scanLoading() {
+    return (
+      <Message icon positive>
+        <Icon name="circle notched" loading />
+        <Message.Content>
+          <Message.Header>Just one second</Message.Header>
+          Checkout in progress.
+        </Message.Content>
+      </Message>
+    );
+  }
+
+  scanError() {
+    return (
+      <Message error>
+        This barcode does not belong to any available physical copy.
+      </Message>
+    );
+  }
+
+  render() {
+    const {
+      headerContent,
+      headerIcon,
+      isOpened,
+      checkoutData,
+      loan,
+    } = this.props;
+    return isOpened ? (
+      <Modal open={isOpened} onClose={this.closeModal} closeIcon size="small">
+        <Header icon={headerIcon} content={headerContent} />
+        <Modal.Content>
+          <ItemsSearch
+            isCheckout
+            checkoutData={checkoutData}
+            loan={loan}
+            loadingHandler={this.isLoadingHandler}
+            errorHandler={this.wrongBarcodeHandler}
+          />
+        </Modal.Content>
+        <Modal.Description>
+          {this.state.isLoading
+            ? this.scanLoading()
+            : this.state.wrongBarcode
+            ? this.scanError()
+            : null}
```

Comment:
I would move this into `<Modal.Content>`, it looks weird without the margin.

(proposal)
![image](https://user-images.githubusercontent.com/9027075/88793737-9c441680-d19d-11ea-88c5-8c8791dad5d3.png)

Reply:
Thanks for noticing (for some reason I did not find it weird until now). Unfortunately the PR has changed and this will not be applied. Thanks anyway!